### PR TITLE
Disable overscan compensation for external displays on iOS

### PR DESCRIPTION
**Pull request by aleksijuvani**
- [x] Tested on all platforms changed
- [x] Compilation warnings were addressed
- [x] `cargo fmt` has been run on this branch
- [x] Added an entry to `CHANGELOG.md` if knowledge of this change could be valuable to users
- [ ] Updated documentation to reflect any user-facing changes, including notes of platform-specific behavior
- [ ] Created or updated an example program if it would help users understand this functionality
- [ ] Updated [feature matrix](https://github.com/rust-windowing/winit/blob/master/FEATURES.md), if new features were added or implemented

r? @mtak-

### CODE REVIEW

File: src/platform_impl/ios/window.rs [hunk 1]
```diff
@@ -165,7 +165,7 @@ impl Inner {
             let uiscreen = match monitor {
                 Some(Fullscreen::Exclusive(video_mode)) => {
                     let uiscreen = video_mode.video_mode.monitor.ui_screen() as id;
-                    let () = msg_send![uiscreen, setCurrentMode: video_mode.video_mode.screen_mode];
+                    msg_send![uiscreen, setCurrentMode: video_mode.video_mode.screen_mode];
```
Comment (mtak-) [inline src/platform_impl/ios/window.rs]: `winit` has been using explicit return types to guard against `msg_send!` calls deducing to the `!` type after never is stabilized (presently they deduce to `()` when the type is not explicitly stated). Previous attempts to stabilize `!` have caused many bugs in winit. See https://github.com/rust-windowing/winit/pull/428 and the other github activity referencing it. If/when never is stabilized, these calls are likely to deduce to `!` causing UB.

I'll open an issue on `gfx` as well, as I know they also have unconstrained types stemming from `msg_send`.

Comment (mtak-) [reply src/platform_impl/ios/window.rs]: https://github.com/gfx-rs/gfx/issues/2939

Comment (aleksijuvani) [reply src/platform_impl/ios/window.rs]: Thank you! I've addressed the issue here.

**Comment by mtak-** [comment]
Tested on AppleTV. Works. Thanks!

**Comment by aleksijuvani** [comment]
Thank you for testing this! I've rebased this against `master`.

**Pull request merged**